client/src/components/common/ArtistHeaderSection.tsx

```diff
@@ -155,8 +167,13 @@ const ArtistHeaderSection: React.FC<{ artist: Artist; isManage?: boolean }> = ({
                 )}
               </div>
             </HeaderDiv>
-            <div className={css` filter: brightness(75%); opacity: .8;`}>
-            <MarkdownContent content={artist.bio}/>
+            <div
```

Comment:
If this is going to be the case for all `<MarkdownContent {...}` does it make sense to just have this set as the CSS in the `MarkdownContent` component?

Reply:
It's just for the artist bio! It could be a variable class though for muted text I think we don't have that in style yet.